src/test_application/activescenes/scenarios_enginetest.cpp

```diff
@@ -69,27 +69,47 @@ constexpr int const gc_maxMaterials = 2;
  */
 struct EngineTestScene
 {
+    ~EngineTestScene()
+    {
+        osp::active::SysRender::clear_owners(m_drawing);
+        osp::active::SysRender::clear_resource_owners(m_drawingRes, *m_pResources);
+        m_drawing.m_meshRefCounts.ref_release(m_meshCube);
+    }
+
+    osp::Resources *m_pResources;
+
     // ID registry generates entity IDs, and keeps track of which ones exist
     lgrn::IdRegistry<osp::active::ActiveEnt> m_activeIds;
 
     // Components and supporting data structures
     osp::active::ACtxBasic          m_basic;    
     osp::active::ACtxDrawing        m_drawing;
+    osp::active::ACtxDrawingRes     m_drawingRes;
 
     // Hierarchy root, needs to exist so all hierarchy entities are connected
     osp::active::ActiveEnt          m_hierRoot;
 
     // The rotating cube
     osp::active::ActiveEnt          m_cube;
```

Comment:
Code scanning
Code scanning

Variable 'testapp::enginetest::EngineTestScene::m_cube' is uninitialized. Always initialize a member variable (type.6).

Warning
Variable 'testapp::enginetest::EngineTestScene::m_cube' is uninitialized. Always initialize a member variable (type.6).